ProjectEureka/project-eureka-flutter
# PE-92 | Migrate to Flutter v2

**What I Did:**
- Updated our codebase so it can work on Flutter v2
- Changed all of our services so it will use HTTPS

*How to Test** 
1. Run `flutter upgrade` on you system. Then `flutter doctor` to verify that you're on the new version.
2. Run ` flutter pub upgrade` to get the newest versions of our packages. Then `flutter pub get` to download the packages.
3. Update the `HOST=""` inside our .env to the proper link found inside our Discord PR channel.
4. Test everything and make sure it still works.
